app/src/main/java/com/nervousfish/nervousfish/activities/MainActivity.java

```diff
@@ -92,7 +93,16 @@ protected void onCreate(final Bundle savedInstanceState) {
 
         sortOnName();
 
-        this.serviceLocator.getBluetoothHandler().start();
+        try {
+            this.serviceLocator.getBluetoothHandler().start();
+        } catch (NoBluetoothException e) {
+            LOGGER.info("Bluetooth not available on device, hiding button");
```

Comment:
I don't think it's necessary as well, but if you put
`new SweetAlertDialog(this, SweetAlertDialog.WARNING_TYPE)
                    .setTitleText(getString(R.string.foo))
                    .setContentText(getString(R.string.bar))
                    .setConfirmText(getString(R.string.dialog_ok))
                    .setConfirmClickListener(null)
                    .show();`
in it, then the user doesn't even have to remember that he doesn't have Bluetooth